tock/hours/forms.py

```diff
@@ -117,7 +119,7 @@ def clean(self):
             # right.
             raise forms.ValidationError('You must report exactly 40 hours.')
 
-        return self.cleaned_data
+        return hasattr(self, 'cleaned_data') and self.cleaned_data or None
```

Comment:
A little more concise: `getattr(self, 'cleaned_data', None)`. Surprised this is needed, though...